eng/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/Microsoft.DotNet.SourceBuild.Tasks.XPlat.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <OutputPath>$(XPlatTasksBinDir)</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build">
+      <Version>15.7.179</Version>
```

Comment:
Would it make sense to encapsulate this version in versions.props?  It would make updating easier and follow the arcade patterns.

Reply:
I'll log a separate issue for that since it should be done for all the other relevant external repos as well.

Reply:
https://github.com/dotnet/source-build/issues/3897